### ISSUE TITLE
original roms?

### ISSUE THREAD

**Issue by rampa069** [trcwm/Speech256#1]
Hi. i am playing  with your syntetizer and was wondering if is possible to change the rom to the original chip. 

 is there any chance?

**Comment by trcwm** [comment]
Hi @rampa069, I'm not sure I understand your question correctly. Do you want to replace the internal ROM in the FPGA with new content or use an external original SP0256 ROM with the FPGA?

Regards,
Niels.

**Comment by rampa069** [comment]
I want to use original SP0236 roms. I have your implementation working in a Oddysey2/Videopac fpga implementation and wat to use the original roms from the real oddysey hardware. is this possible?

Thanks in advance.

**Comment by trcwm** [comment]
Yes, that should be possible. If you dump the SP0236, name it 'ctrlrom.hex' and use genctrlrom.py to generate a new rom for the FPGA, there is a chance it could work.

**Comment by rampa069** [comment]
dont seem the same format. your rom seem to be 4k (converting the hex to binary)  the "original" rom from the oddysey2 is 2K in size (sp0256b-019.bin) and 16k the extended (spr128-003.bin).  the python  program fails with this roms.

**Comment by trcwm** [comment]
Ah, the Python script and the Verilog code will need adapting to different ROM sizes. I can investigate when I have some free time, but I don't expect that will happen very soon.

**Comment by rampa069** [comment]
thank you very much. i'll have a look to the python core to try to figure how to change.
